test/regressions/site/src/tests/Input/Inputs.js

```diff
@@ -9,10 +9,15 @@ const styleSheet = createStyleSheet('Inputs', () => ({
   container: {
     display: 'flex',
     flexDirection: 'column',
+    width: 200,
   },
   input: {
     margin: 10,
   },
+  largeInput: {
+    margin: 10,
+    width: 300,
```

Comment:
I would omit `largeInput` here and use the same `input` class.  Add it to the `outer: { width: 300 }` to the outermost `<div/>`, it would better prove that the width fills the given space.